packages/xchain-radix/src/client.ts

```diff
@@ -0,0 +1,566 @@
+    const transactionBinary = new Uint8Array(binaryString.split('').map((char) => char.charCodeAt(0)))
+    const transactionSummary = await LTSRadixEngineToolkit.Transaction.summarizeTransaction(transactionBinary)
+    const withdrawAccount = Object.keys(transactionSummary.withdraws)[0]
+    const depositAccount = Object.keys(transactionSummary.deposits)[0]
+    const withdrawResource = Object.keys(transactionSummary.withdraws[withdrawAccount])[0]
+    const withdrawAmount: number = transactionSummary.withdraws[withdrawAccount][withdrawResource].toNumber()
+
+    const transaction: Tx = {
+      from: [
+        {
+          from: withdrawAccount,
+          amount: baseAmount(withdrawAmount),
+          asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+        },
+      ],
+      to: [
+        {
+          to: depositAccount,
+          amount: baseAmount(withdrawAmount),
+          asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+        },
+      ],
+      date: confirmed_at,
+      type: TxType.Transfer,
+      hash: intent_hash,
+      asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+    }
+    return transaction
+  }
+
+  /**
+   * Creates a transaction using the SimpleTransactionBuilder, signs it with the
+   * private key and returns the signed hex
+   * @param params - The transactions params
+   * @returns A signed transaction hex
+   */
+  async transfer(params: TxParams): Promise<string> {
+    const networkId = this.getRadixNetwork()
+    const radixPrivateKey = this.getRadixPrivateKey()
+
+    if (params.asset == undefined) {
+      throw new Error('asset can not be undefined')
+    }
+    const preparedTransaction = await this.prepareTx(params)
+    const manifest: TransactionManifest = {
+      instructions: { kind: 'String', value: preparedTransaction.rawUnsignedTx },
+      blobs: [],
+    }
+    try {
+      const gatewayStatusResponse: GatewayStatusResponse = await this.gatewayApiClient.status.getCurrent()
+      const notarizedTransaction = await TransactionBuilder.new().then((builder) =>
+        builder
+          .header({
+            networkId: networkId,
+            startEpochInclusive: gatewayStatusResponse.ledger_state.epoch,
+            endEpochExclusive: gatewayStatusResponse.ledger_state.epoch + 10,
+            nonce: generateRandomNonce(),
+            notaryPublicKey: radixPrivateKey.publicKey(),
+            notaryIsSignatory: true,
+            tipPercentage: 0,
+          })
+          .manifest(manifest)
+          .notarize(radixPrivateKey),
+      )
+
+      const compiledTransaction = await RadixEngineToolkit.NotarizedTransaction.compile(notarizedTransaction)
+      const compiledTransactionHex = Convert.Uint8Array.toHexString(compiledTransaction)
+      await this.broadcastTx(compiledTransactionHex)
```

Comment:
Once we change `broadcastTx` to return the hash we can just return its response at this point.

Reply:
I'm returning the transaction id instead. would that be ok

Reply:
Well both `broadcastTx` and `transfer` should return the tx ID